tests/basic_correctness/test_basic_server_correctness.py

```diff
@@ -0,0 +1,239 @@
+import asyncio
+from os import getenv
+from typing import Dict, List, Type
+
+import openai
+import pytest
+import torch
+from datasets import load_dataset
+from openai import AsyncOpenAI
+from transformers import AutoTokenizer
+
+from tests.conftest import HfRunnerNM
+from tests.models.compare_utils import check_logprobs_close
+from tests.utils.logging import make_logger
+from tests.utils.server import ServerContext
+from vllm.model_executor.layers.quantization import get_quantization_config
+
+
+@pytest.fixture(scope="session")
+def client():
+    client = openai.AsyncOpenAI(
+        base_url="http://localhost:8000/v1",
+        api_key="token-abc123",
+    )
+    yield client
+
+
+@pytest.fixture
+def hf_runner_nm() -> Type[HfRunnerNM]:
+    return HfRunnerNM
+
+
+async def my_chat(
+    client,
+    model: str,
+    messages: List[Dict],
+    max_tokens: int,
+    temperature: float,
+    num_logprobs: int,
+):
+    """ submit a single prompt chat and collect results. """
+    return await client.chat.completions.create(model=model,
+                                                messages=messages,
+                                                max_tokens=max_tokens,
+                                                temperature=temperature,
+                                                logprobs=True,
+                                                top_logprobs=num_logprobs)
+
+
+@pytest.mark.parametrize(
+    "model, max_model_len, sparsity, gptq_config",
+    [
+        ("mistralai/Mistral-7B-Instruct-v0.2", 4096, None, None),
+        # pytest.param("mistralai/Mixtral-8x7B-Instruct-v0.1", 4096, None, None,
```

Comment:
I was going to mark them as skipped, pointing to asana tickets, but since this is probably going upstream I shouldn't  be pointing to an internal system.  I'm removing the entries altogether, and specifying how to add them in the open bug reports.

Reply:
yeah, just remove them. we can add them later as we sort through blocking issues.